www/templates/html/Peoplefinder/Developers.tpl.php

```diff
@@ -1,174 +1,119 @@
-<style type="text/css">
-    blockquote {
-         background: none repeat scroll 0pt 0pt rgb(240, 240, 240);
-         margin: 15px 0pt;
-         padding: 10px;
-         width: auto;
-    }
+<?php
+$page->addHeadLink('https://cdn.jsdelivr.net/highlight.js/9.2.0/styles/solarized-dark.min.css', 'stylesheet');
+?>
 
-    blockquote > p {
-         clear: none;
-         margin: 0pt;
-         padding: 0pt;
-    }
-
-    div.resource {
-         border-bottom: #F0F0F0 5px solid;
-         margin-bottom: 20px;
-    }
-
-    #maincontent div.resource > ul {
-        padding-left:0;
-    }
-
-    div.resource > ul > li {
-        list-style:none;
-    }
-
-    a.resources
-    {
-        float:right;
-        font-size:12px
-    }
-</style>
-
-<script type="text/javascript">jQuery = $ = WDN.jQuery;</script>
-
-<script type="text/javascript" src="scripts/jquery.beautyOfCode.js"></script>
-
-<script type="text/javascript">
-    $.beautyOfCode.init({
-        theme: "RDark",
-        brushes: ['Xml', 'JScript', 'CSharp', 'Plain', 'Php', "Java", "JavaFX"],
-        ready: function() {
-            $.beautyOfCode.beautifyAll();
-
-        }
-    });
-</script>
-
-<div class="three_col left">
-
-    <?php
+<div class="wdn-grid-set">
+    <div class="bp3-wdn-col-three-fourths">
+        <?php
         $resource = "UNL_Peoplefinder_Developers_" . $context->resource;
         $resource = new $resource;
         ?>
         <div class="resource">
-        <h1 id="instance" class="sec_main"><?php echo $resource->title; ?> Resource</h1>
-        <h3>Details</h3>
-        <ul>
-            <li>
-                <h4 id="instance-uri"><a href="#instance-uri">Resource URI</a></h4>
-                <blockquote>
+            <h1 id="instance" class="sec_main">API: <?php echo $resource->getTitle(); ?> Resource</h1>
+
+            <h2 id="instance-uri"><a href="#instance-uri">Resource URI</a></h2>
```

Comment:
I believe these were there to help with linking. If a visitor wanted to link to a certain part of the documentation, the `<a>` would make it easier. However, I don't think they are actually very helpful, so I will remove them.